app/models/user.rb

```diff
@@ -1,6 +1,10 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please in order for the user to confirm his/her email, `:confirmable` has to be enabled in the `User model`, please kindly fix that. See screenshot below for clarifications and kindly fix that.

![confirmation](https://user-images.githubusercontent.com/62849872/156046434-56454cc7-c85f-4a82-8ae9-13e109144f78.png)

Reply:
Noted sir, Added confirmable.